lisa/tools/python.py

```diff
@@ -52,6 +54,8 @@ def dependencies(self) -> List[Type[Tool]]:
 
     def _install(self) -> bool:
         package_name = "python3-pip"
+        if isinstance(self.node.os, BSD):
+            package_name = "py39-pip"
```

Comment:
Is there a version for all py3? py39 may have compatibility issue with newer python versions like 3.10, 3.11.

Reply:
This is the most recent version of pip I saw available through the pkg utility. I will look into whether or not there is a generic one.

Reply:
Changed it to installing devel/py-pip so that it is not version dependent.